jwst/extract_1d/ifu.py

```diff
@@ -1080,3 +1124,54 @@ def shift_ref_image(mask, delta_y, delta_x, fill=0):
     temp[..., oslice_y, oslice_x] = mask[..., islice_y, islice_x]
 
     return temp
+
+def sigma_clip_extended_region(data, mask_targ, wmap, sigma_clip):
+    """ Convert 2D back mask to 3D and flag pixels identified as outliers with 0
+        For each wavelength plane perform sigma clipping of extracted region
+
+    Parameters
+    ----------
+    data: 3-D
+        Input data array to perform extraction from
+
+    mask_targ : ndarray, 2-D or 3-D
+        Mask of pixels defining the extended source region. A value of 1 indicated
+        pixel is in the extraction region.
+    wmap: ndarray, 3-D
+       weight map for IFU
+    sigma_clip : float
+        outlier sigma clipping parameter
+
+    Returns
+    -------
+    temp : ndarray, same type and shape as `mask_bkg`
+        A copy of `mask_bkg`, but outliers flagged as 0
```

Comment:
This docstring doesn't coincide with what the function actually returns.

Reply:
whoops - I fixed that